src/fu-usb-device.c

```diff
@@ -130,10 +130,9 @@ fu_usb_device_open (FuDevice *device, GError **error)
 		if (idx != 0x00) {
 			g_autofree gchar *tmp = NULL;
 			tmp = g_usb_device_get_string_descriptor (priv->usb_device,
-								  idx, error);
-			if (tmp == NULL)
-				return FALSE;
-			fu_device_set_vendor (device, g_strchomp (tmp));
+								  idx, NULL);
```

Comment:
don't we want to do an error local dance here and at least notate to debug logs that the string failed to read and why?  It could be indicative of a hardware problem and this could help debugging later.

Reply:
That is a good idea, let me update the patch.

Reply:
Pushed an update, let me know what you think

Reply:
This is how it looks like in the logs:
```
17:24:52:0251 FuDevice             using 0703a244275ed772f8e276c4b31389afc296101d for usb:00:03
17:24:53:0251 FuUsbDevice          failed to load product string for usb device 1:30: USB error on device 413c:81e1 : Operation timed out [-7]
17:24:53:0275 FuPluginFastboot     fastboot version=0.5
17:24:53:0335 FuPlugin             emit added from fastboot: 0703a244275ed772f8e276c4b31389afc296101d
```

Reply:
Looks great, thanks.

Reply:
@hughsie are you fine with these new changes?